docs/docs/concepts/policies.mdx

```diff
@@ -2,5 +2,699 @@
+
+## Testing
+
+The policy is a usual Rasa Policy and can be tested in the same way as any other
+policy.
+
+### Testing interactively
+
+Once trained, you can test your assistant interactively by running the following
+command:
+
+```bash
+rasa shell
+```
+
+If a flow you'd like to implement doesn't already work out of the box, you can
+add try to change the examples for the intentless policy. Don't forget that you
+can also add and edit the traditional Rasa primitives like intents, entities,
+slots, rules, etc. as you normally would. The `IntentlessPolicy` will kick in
+only when the traditional primitives have low confidence.
+
+### End-to-End stories
+
+As part of the beta, we're also releasing a beta version of a new End-To-End
+testing framework. The `rasa test e2e` command allows you to test your bot
+end-to-end, i.e. from the user's perspective. You can use it to test your bot in
+a variety of ways, including testing the `IntentlessPolicy`.
+
+To use the new testing framework, you need to define a set of test cases in a
+test folder, e.g. `tests/e2e_test_stories.yml`. The test cases are defined in a
+similar format as stories are, but contain the user's messages and the bot's
+responses. Here's an example:
+
+```yaml title="tests/e2e_test_stories.yml"
+test_cases:
+  - test_case: transfer charge
+    steps:
+      - user: how can I send money without getting charged?
+      - utter: utter_faq_0
+      - user: not zelle. a normal transfer
+      - utter: utter_faq_7
+```
+
+**Please ensure all your test stories have unique names!** After setting the
+beta feature flag for E2E testing in your current shell with
+`export RASA_PRO_BETA_E2E=true`, you can run the tests with
+`rasa test e2e -f tests/e2e_test_stories.yml`
+
+## Security Considerations
+
+The intentless policy uses the OpenAI API to create responses.
+This means that your users conversations are sent to OpenAI's servers.
+
+The response generated by OpenAI is not send back to the bot's user. However,
+the user can craft messages that will misslead the intentless policy. These
+cases are handled gracefully and fallbacks are triggered.
+
+The prompt used for classification won't be exposed to the user using prompt
+injection. This is because the generated response from the LLM is mapped to
+one of the existing responses from the domain,
+preventing any leakage of the prompt to the user.
+
+More detailed information can be found in Rasa's webinar on
+[LLM Security in the Enterprise](https://info.rasa.com/webinars/llm-security-in-the-enterprise-replay).
+
+## FAQ
+
+### What about entities?
+
+Entities are currently not handled by the intentless policy. They have to still
+be dealt with using the traditional NLU approaches and slots.
+
+### What about custom actions?
+
+At this point, the intentless policy can only predict utterances but not custom
+actions. Triggering custom actions needs to be done by traditional policies,
+such as the rule- or memoization policy.
```

Comment:
this can go, also?

Reply:
added a TODO to rewrite the section on the intentless policy because it plays a specific, narrow role in calm compared to what we first used it for